Lib/test/test_code.py

```diff
@@ -865,6 +865,7 @@ def __init__(self, f, test):
                     self.test = test
                 def run(self):
                     del self.f
+                    gc_collect()
```

Comment:
If I understand right, `gc.collect()` is how a user can make sure all objects that need to be finalized/deleted have been.  Is that clear in the docs?